docs/self-managed/setup/overview.md

```diff
@@ -31,7 +31,7 @@ For details on supported environments (e.g. Java or Elasticsearch versions), see
 
 You have the following options to run or deploy the above components in a self-managed fashion:
 
-- **Helm/Kubernetes**: We strongly recommend using Kubernetes to run Camunda 8 in production. In addition to stock Kubernetes we also officially support variety of providers like Red Hat OpenShift and Amazon EKS. Also using Kubernetes with Minikube or KIND can be an interesting environment to run Camunda 8 locally on developer machines.
+- **Kubernetes**: We strongly recommend using Kubernetes to run Camunda 8 in production. We provide supported HELM charts to deploy Camunda 8 into your clusters. We suport [CNCF Certified Kubernetes distributions](https://www.cncf.io/training/certification/software-conformance/) including a variety of providers like RedHat OpenShift and Amazon EKS. Consider using [minikube](https://minikube.sigs.k8s.io/docs/start/?arch=%2Fmacos%2Fx86-64%2Fstable%2Fbinary+download) or [kind](https://kind.sigs.k8s.io/docs/user/quick-start) to run Camunda 8 locally on developer machines.
```

Comment:
see my comment in https://github.com/camunda/camunda-docs/pull/3888#discussion_r1628107262 - why not to move this contents to `supported-environments.md` for better user experience to find the info on the same page?

Reply:
I see the point, but I would like to reiterate distributions here as well. I removed "support" from this page to focus on how to run rather than on the support statement.